api/provider/aws/environment_update.go

```diff
@@ -6,6 +6,10 @@ import (
 	"github.com/quintilesims/layer0/common/models"
 )
 
+// Update updates an ECS Cluster using the specified Update Environment Request, and
```

Comment:
[nit] I don't think we need to reiterate what is visible in the function signature unless it's not reasonably apparent. "Returns an error" is specifically what made me think of that here --- if we say that, would we need to be more explicit about _when_ it returns an error? Just seems like redundant information.

Reply:
Stating that it returns an error is probably an overkill. I must admit that it felt weird typing that out explicitly, but I wasn't sure if I needed to for consistency's sake.

Reply:
on that note, listing the models' fields seems overkill IMO as well